### DISCUSSION TITLE
Support for long TXT records

### DISCUSSION THREAD

**Discussion by hostep** [spatie/dns#80] [Ideas]
Hi guys, since I'm unable to report issues here, I'm trying this way.

TXT records sometimes can exceed 255 characters. We have some domain names in our portfolio that have SPF records that exceed this max limit.
If it's longer than 255 characters, the record gets split into multiple sections, as explained here: https://support.opensense.com/splitting-a-long-spf-txt-record

Currently this library doesn't seem to handle this, it just takes over the `" "` when getting the value of a TXT record.
Solution seems to be pretty simple, use this line in https://github.com/spatie/dns/blob/83be2fa60fe4634dde54a7bfab66f36d6b640e5a/src/Records/Record.php#L109-L112

```php
return str_replace('" "', '', trim($value, '"'));
```

If you're open to this idea, I'll see if I can send in a PR for this later today/tomorrow when I find some more time.

**Comment by freekmurze** [comment]
I'd welcome a PR that handles this. If possible, also include a test that make sure this work. Feel free to use that long TXT record you need to be handled in that test.

**Reply by hostep** [reply]
Okay great, PR created: https://github.com/spatie/dns/pull/81 🙂
